### DISCUSSION TITLE
原來我們都被Netflix掌握 ?! Netflix 如何運用 Python 拉進與用戶的距離

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#31] [General]
原來我們都被Netflix掌握 ?!

根據 Netflix工程師在 Netflix Tech Blog 揭密，Python被用於跟蹤用戶使用狀況，包含個性化推薦影音的演算法與內容傳遞網路(CDN)，整個內容生命週期全都使用 Python 確保網路傳輸內容。

此外，Netflix也利用 Python 開發機器學習模型，來預測所有影音內容的受眾規模、收視率和其他需求指標的核心。

新聞來源：https://buzzorange.com/techorange/2019/05/27/python-at-netflix/

**Comment by Isongzhe** [comment]
哭阿 難怪每次一打開Netflix就停不下來
